Makefile

```diff
@@ -134,7 +134,7 @@ generate: controller-gen rice
 .PHONY: operator-build
 ## Build the operator image
 operator-build: manager
-	$(CONTAINER_TOOL) build -t $(IMG) .
+	$(CONTAINER_TOOL) build --build-arg VERSION=$(VERSION) -t $(IMG) .
```

Comment:
Why is this necessary?

Reply:
There is a `VERSION` argument in the Dockerfile that is not set and it does not have a default value. 
Isn't an issue?

Podman does not like it
```
$ make operator-build CONTAINER_TOOL=podman
/home/pedro/Work/infinispan-operator/bin/controller-gen object paths="./..."
/home/pedro/Work/infinispan-operator/bin/rice embed-go -i controllers/dependencies.go -i controllers/grafana.go
find . -type f -name 'rice-box.go' -exec sed -i "s|time.Unix(.*, 0)|time.Unix(1620137619, 0)|" {} \;
go fmt ./...
go vet ./...
go build -o bin/manager main.go
podman build -t quay.io/infinispan/operator:latest .
WARN[0001] missing "VERSION" build argument. Try adding "--build-arg VERSION=<VALUE>" to the command line 
```

Reply:
actually, `VERSION` is set in the base image.
Adding `RUN echo ${VERSION}` I got (with docker)
```
Step 1/11 : FROM registry.access.redhat.com/ubi8/go-toolset:1.15.14 AS build
...
Step 3/11 : RUN echo ${VERSION}
 ---> Running in 6aeb06d29666
1.15.14
```
Is it intentional?

Reply:
> There is a `VERSION` argument in the Dockerfile that is not set and it does not have a default value.
> Isn't an issue?

No it is, good spot :+1: 

> actually, `VERSION` is set in the base image. Adding `RUN echo ${VERSION}` I got (with docker)
> Is it intentional?

Nope :sweat_smile: